### ISSUE TITLE
Add to right click service menu [implemented and needs testers]

### ISSUE THREAD

**Issue by adammoisa** [chrissimpkins/Crunch#9]

**Comment by chrissimpkins** [comment]
Thank you very much for the suggestion!  Will look into it.

**Comment by chrissimpkins** [comment]
I added support for a new `Crunch Image(s)` macOS service that works with one or more PNG files from the right click menu.  It does not spawn the GUI (which is simply a receptacle for file drops anyways), simply executes the scripted approach to PNG file optimizations. I can verify that it works on macOS 10.12.6.

Would greatly appreciate it if you and/or anyone else with an interest would be willing to install it and give it a try.  It is currently located in this branch:

https://github.com/chrissimpkins/Crunch/tree/macos-service

and you can install the service with the new Makefile in that branch using the following command from the root of the repository:

```
$ make install-service
``` 

This request copies the *.workflow directory that defines the service from the new `service/Crunch\ Image\(s\).workflow` directory to the necessary install path `~/Library/Services/Crunch\ Image\(s\).workflow`.  This requires sudo permissions in order to copy to that directory on your system.  You will be prompted for a password.  If you prefer to move it yourself, you can run the following in your terminal from the root of the repository:

```
$ sudo cp -R service/Crunch\ Image\(s\).workflow ~/Library/Services/Crunch\ Image\(s\).workflow
```

Once the copy takes place, you should be able to access the service through the right click menu on an image file > Services > Crunch Image(s) in the Finder when you select any PNG image. There is a spinning gear in the status bar as the image is processed and it is cleared when the execution of the script is complete.

If you want to uninstall the service use this from the root of the repository:

```
$ make uninstall-service
```

or this command in your terminal in any working directory:

```
$ sudo rm -rf ~/Library/Services/Crunch\ Image\(s\).workflow
```

If this seems to address this request adequately, we will push these changes as v1.1.0.

**Comment by chrissimpkins** [comment]
Detailed documentation on the new macOS right click service is now available in https://github.com/chrissimpkins/Crunch/blob/macos-service/docs/SERVICE.md

**Comment by chrissimpkins** [comment]
This new macOS service will be included in the v1.1.0 release of Crunch.

**Comment by chrissimpkins** [comment]
Release v1.1.0 is now available in Releases and includes support for a new right click menu macOS service.  See README for installation instructions.  Pushing new version to Homebrew cask.  It should be available by tomorrow.
